kbase-extension/static/kbase/js/widgets/narrative_core/staticNarrativesManager.js

```diff
@@ -11,7 +11,7 @@ define([
     'common/runtime',
     'util/display',
     'util/timeFormat',
-    'kbase-generic-client-api',
+    'kb_common/jsonRpc/dynamicServiceClient',
```

Comment:
swap out the other client for this one, as it's more stable.

Reply:
is there any difference between using `kb_common` and `common`? Can we standardise and use just one?

Reply:
Yeah, they're different.
`common` is the directory under `kbase-extension/static/kbase/js/` with modules that are "common" between the nbextensions and the widgets that are under `kbase-extension`.

`kb_common` is a separate package that's installed from uh, bower, I think right now. But it's in the kbase github.

Reply:
Ah, OK. Thanks for clarifying that!